app/renderer/js/spellchecker.js

```diff
@@ -2,6 +2,12 @@
 
 const { SpellCheckHandler, ContextMenuListener, ContextMenuBuilder } = require('electron-spellchecker');
 
+const LanguageDetect = require('languagedetect');
+
+const lngDetector = new LanguageDetect();
+
+const ISO6391 = require('iso-639-1');
```

Comment:
If you've added new `npm` modules here, please also add those to your `package.json` and push the changes.